src/Zicht/Bundle/MenuBundle/StatusProvider/ValidateNestedTreeProvider.php

```diff
@@ -8,29 +8,41 @@
 
 use Symfony\Bridge\Doctrine\RegistryInterface;
 use Zicht\Bundle\MenuBundle\Entity\MenuItem;
+use Zicht\Bundle\StatusBundle\StatusProvider\StatusProviderHelper;
 use Zicht\Bundle\StatusBundle\StatusProvider\StatusProviderInterface;
 
 /**
  * Class ValidateNestedTreeProvider
  *
  * @package Zicht\Bundle\MenuBundle\StatusProvider
  */
-class ValidateNestedTreeProvider implements StatusProviderInterface
+class ValidateNestedTreeProvider extends StatusProviderHelper implements StatusProviderInterface
 {
     /** @var boolean */
     protected $isValid;
 
     /** @var array */
     protected $values;
 
+    /** @var RegistryInterface */
+    protected $doctrine;
+
     /**
      * ValidateNestedTreeProvider constructor.
      *
      * @param RegistryInterface $doctrine
      */
     public function __construct(RegistryInterface $doctrine)
     {
-        $repository = $doctrine->getRepository(MenuItem::class);
+        $this->doctrine = $doctrine;
+    }
+
+    /**
+     * @{inheritDoc}
+     */
+    protected function check()
```

Comment:
De check methode verwacht dat er een `[$isValid, $valuesArray]` response is, deze response wordt door `StatusProviderHelper` aan de variabelen toegekend.  Dat moet dus niet gebeuren zoals hieronder in de methode plaatsvind.

Zie https://github.com/zicht/status-bundle/blob/release/2.x/src/Zicht/Bundle/StatusBundle/StatusProvider/StatusProviderHelper.php#L27